### PR TITLE
Added Eviction Policy Property for VirtualMachineScaleSetVMProfile for version 2017-12-01

### PR DESCRIPTION
**Pull request by mahajanudit-microsoft-zz**
EvictionPolicy can take 2 possible values: Deallocate, Delete
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by mahajanudit-microsoft** [comment]
There is only 1 commit in this PR, added EvictionPolicy

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json [hunk 1]
```diff
@@ -6915,6 +6915,18 @@
...
+        "evictionPolicy": {
+          "type": "string",
+          "description": "Specifies the eviction policy for virtual machines in the low priority scale set. <br><br>Minimum api-version: 2017-10-30-preview",
+          "enum": [
+            "Deallocate",
+            "Delete"
+          ],
+          "x-ms-enum": {
+            "name": "VirtualMachineEvictionPolicyTypes",
+            "modelAsString": false
```
Comment (dsgouda) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json]: Marking `modelAsString` as false can cause breaking changes if new enum types are added. Please reconsider this.

**Review by dsgouda — COMMENTED** [review]
Looks good other than the minor comment

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json (hunk 1 above)
Comment (mahajanudit-microsoft) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json]: Ok, I will change it to true then

Comment (mahajanudit-microsoft) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json]: Done

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**